<launch>
 <node pkg="pathgo_imu" type="sync_imu_node" name="pathgo_imu_node" output="screen" respawn="true"> 
      <rosparam file="$(find pathgo_imu)/params/imu_params.yaml" command="load"/>
 </node> 
 <node pkg="tf" type="static_transform_publisher" name="imu_broadcaster" args="0 0 0 0 0 0 base_footprint imu_base 100" />
</launch>
